<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BzMainWindow</class>
 <widget class="QMainWindow" name="BzMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BzMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="BzView" name="bzView" native="true">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionCharger"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionQuitter"/>
    <addaction name="actionEffacer"/>
    <addaction name="actionConstruire"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOutils"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEffacer"/>
   <addaction name="actionConstruire"/>
   <addaction name="actionQuitter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau">
   <property name="icon">
    <iconset resource="bzmainwindow.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="icon">
    <iconset resource="bzmainwindow.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="icon">
    <iconset resource="bzmainwindow.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="bzmainwindow.qrc">
     <normaloff>:/images/Quit.png</normaloff>:/images/Quit.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionEffacer">
   <property name="icon">
    <iconset resource="bzmainwindow.qrc">
     <normaloff>:/images/Redo.png</normaloff>:/images/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Effacer</string>
   </property>
  </action>
  <action name="actionConstruire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="bzmainwindow.qrc">
     <normaloff>:/images/BezierConstruct.png</normaloff>:/images/BezierConstruct.png</iconset>
   </property>
   <property name="text">
    <string>Construire</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BzView</class>
   <extends>QWidget</extends>
   <header>bzview.h</header>
   <container>1</container>
   <slots>
    <signal>mousePos(QPoint)</signal>
    <slot>on_Effacer()</slot>
    <slot>on_Construire(bool)</slot>
    <slot>updateStatus(QPoint)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bzmainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionConstruire</sender>
   <signal>toggled(bool)</signal>
   <receiver>bzView</receiver>
   <slot>on_Construire(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEffacer</sender>
   <signal>triggered()</signal>
   <receiver>bzView</receiver>
   <slot>on_Effacer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>BzMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
